pkg/daemon/kernelargs.go

```diff
@@ -72,6 +72,10 @@ func HostUsesNetworkingInInitrd() (bool, error) {
 	return isArgInUse("rd.neednet", "")
 }
 
+func HostDisablesPredictableNicNames() (bool, error) {
+	return isArgInUse("net.ifnames=0", "")
```

Comment:
I still feel like this is exactly the kind of detail that should live in nmstate, not here.

And it looks like that already happened in https://github.com/nmstate/nmstate/commit/c9fd1e80d3c87e1f844edcd86e9d36ae499ce717